habbitbuddy/screens/buddies.js

```diff
@@ -6,7 +6,9 @@ import Card from '../shared/card';
 export default function Buddies({ navigation }) {
     const [reviews, setReviews] = useState([
 
-        {name: 'Andrew Baker', habit: 'Attending chapel', hobby: 'Reading', key: '1'}
```

Comment:
it looks like this is the same line of code except that one version is missing the comma at the end